cloudinit/distros/__init__.py

```diff
@@ -936,6 +936,9 @@ def set_keymap(self, layout, model, variant, options):
         else:
             raise NotImplementedError()
 
+    def _get_tmp_exec_path(self) -> str:
```

Comment:
Given that this interface is exposed to other packages, consider dropping the leading `_`?

Reply:
Done.